parcels/tools/converters.py

```diff
@@ -6,6 +6,7 @@
 import cftime
 import numpy as np
 import xarray as xr
+from numpy.typing import ArrayLike, NDArray
```

Comment:
Why not use `import numpy.typing as npt` like in grid.py? Better to stay consistent throughout Parcels?